### PR TITLE
Form collaboration

### PR DESCRIPTION
**Pull request by tahiry-dev**
Our first pull request for the project HTML FORM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    </header>
+    <div class="main-logo flex-display">
+      <a href="#"
+        ><img class="main-logo-image" src="images/intuit_logo.png" alt="Intuit"
+      /></a>
+    </div>
+    <div class="small-logos flex-display">
+      <div>
+        <a href="#"
+          ><img class="mint-logo" src="images/mint_logo.png" alt="mint"
```
Comment (SpaYco) [inline index.html]: * [ ] It shows the Mint logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        /></a>
+      </div>
+      <div>
+        <a href="#"
+          ><img
+            class="quickboks-logo"
+            src="images/quickbooks_logo.png"
+            alt="quickboks"
+        /></a>
```
Comment (SpaYco) [inline index.html]: * [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      </div>
+      <div>
+        <a href="#"
+          ><img
+            class="turbotax-logo"
+            src="images/turbotax_logo.png"
+            alt="turbotax"
+        /></a>
```
Comment (SpaYco) [inline index.html]: * [ ] It shows the Turbotax logo and text (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 

Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [ ] Files not related to the project should not be committed and added to .gitignore ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- **I attached more required changes below**



### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Great work on this project :100: 

However, you did not set up a linter. I'm sorry that this was not previously mentioned in the last review, but this is quite important for your future projects. Please exercise patience :pray: 

- [ ] You need to set up GitHub actions as a linter

- [ ] Also, remove the readme from the git ignore, and also delete that unnecessary notes.txt

For more info, check [here](https://github.com/microverseinc/linters-config/tree/master/html-css)

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job so far team :clap: :+1: :100: but you need to make some minor changes :hammer: :wrench: :toolbox:  in order to get your project approved :checkered_flag: 

- [ ] Fix the linter errors:
image

- [ ] Remove README.md from the .gitignore file.

- [ ] Give to your form a fixed width instead of percentages.

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved! 

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**